vtex-apps/psv2-webhook
# add excludedAuthors list

#### What is the purpose of this pull request?

Enable to ignore events from a specific author at the weebhook.

#### What problem is this solving?

Overflow of events to the external integration due to events sent for the profile created by the ffm order at the Post Purchase Flow.

#### How to test it?

Install the beta version: `vtex.psv2-webhook@0.0.7-beta.0`  and you should not receive events from the author `7f4f1a89-5974-4841-a045-c417f2a584a6`

<img width="973" alt="image" src="https://github.com/user-attachments/assets/e0e8ce84-098e-45f3-8440-e48881f687c3">

Env for test: https://webhook--dunnesstoresqa.myvtex.com/admin/apps/vtex.psv2-webhook@0.0.x/setup

#### How does this PR make you feel? [:link:](http://giphy.com/)



![](https://i.giphy.com/media/v1.Y2lkPTc5MGI3NjExb3IzNWk0Ync5dWp6bzM2MGpqejRtam16MWJkd2hrMms4Z2pjNXJ4byZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/IdmfEtnMWPzOg/giphy-downsized.gif)

#### Types of changes
